src/data/examples/exclusivity-strategy.json

```
@@ -0,0 +1,11 @@
+{
+  "default": "none",
+  "tokens": {
+    "USDT": {
```

I realised that we don't have good widespread support for USDT on the chains that currently support depositExclusive, so it limits our ability to test. WETH -> Polygon should be fine to test with.

```suggestion
    "WETH": {
```